docs/docs/tutorials/creating_main.go.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Creating main.go
+parent: Tutorials
+nav_order: 4
+---
+
+## Creating main.go
+
+Before creating `main.go` file we are assuming that the user has created a `go.mod` with the name `github.com/myuser/myrepo`. Then import here is using the path to the generated files in `proto/helloworld` relative to the root of the repository.
```

Comment:
If we add a section about creating a new module on the first page, we can remove that section from this document.